general/quick-start.md

```diff
@@ -7,17 +7,23 @@ title: Quick Start
 
 1. Install Jellyfin on [your system](xref:admin-installing) with the installation method for your platform.
 
-2. Browse to `http://127.0.0.1:8096` to access the included web client.
+1. [OPTIONAL from 10.7.0 onwards] Edit the Jellyfin web configuration (`/usr/share/jellyfin/web/config.json` on Debuntu, or `/jellyfin/jellyfin-web/config.json` in Docker) and adjust the options to fit your desired privacy level; our defaults sacrifice some absolute self-hosting for well-requested features.
```

Comment:
as /jellyfin/jellyfin-web/config.json will get wiped on container recreation, this will be a problem for docker.

Reply:
It's a problem for Debian too actually. Let me reword this.